src/api/routers/participantsRouter.ts

```diff
@@ -327,15 +339,14 @@ export function createParticipantsRouter() {
 
       const editedKey = await getApiKey(participant.siteId, keyId);
       if (!editedKey) {
-        return res.status(404).send('KeyId was invalid');
+        return res.status(404).send('SiteId was invalid');
```

Comment:
KeyId invalid not the correct error response here - if keyId is invalid, it will not even hit this line of code, but is SiteId invalid, it will hit this code